src/config/theme.ts

```diff
@@ -126,6 +139,14 @@ export const baseThemeDef: ThemeOptions = {
     },
     links: '#1976D2',
     activeStatus: '#75559F',
+    grayscale: {
+      main: '#6E6E6E',
+      dark: '#4D4D4D',
+      contrastText: '#fff',
+      p30: alpha('#6E6E6E', 0.3),
+      p12: alpha('#6E6E6E', 0.12),
+      p08: alpha('#6E6E6E', 0.08),
+    },
```

Comment:
Review w/ Bobby - this is how things appear specified in figma, so trying to match that